Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 Due to package verification, the latest version below is the unpublished version and the date is meaningless.
 however, it has to be formatted properly to pass verification tests.
 
+## [Unreleased]
+
+### Fixed
+
+#### Actions
```

Comment:
Does Fixed need a sub category for Actions? Will we have multiple fixes in this area?

Reply:
Not sure it's strictly necessary. There will definitely be multiple entries in this area. My initial thought behind having an explicit subheading for actions was that most of the fixes elsewhere tended to not be that relevant to most users whereas the action-related changes/fixes tend to be the most important ones.